twingate/test_utils.go

```diff
@@ -0,0 +1,36 @@
+package twingate
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const WaitDuration = 500 * time.Millisecond
+
+func WaitTestFunc() resource.TestCheckFunc {
+	return func(state *terraform.State) error {
+		// Sleep 500 ms
+		time.Sleep(WaitDuration)
+
+		return nil
+	}
+}
+
+func ComposeTestCheckFunc(fs ...resource.TestCheckFunc) resource.TestCheckFunc { //nolint:varnamelen
+	return func(state *terraform.State) error {
+		if err := WaitTestFunc()(state); err != nil {
```

Comment:
not sure it's needed, as wait func never returns error

Reply:
Just for code completeness. If anyone ever goes and changes that func and doesn't look here